modules/_labs/select/react/lib/SelectBase.tsx

```diff
@@ -192,25 +185,46 @@ const SelectButton = styled('button')<Pick<SelectBaseProps, 'error' | 'grow' | '
       },
     },
   },
-  ({error, isMenuHidden}) => {
+  ({theme}) => {
+    const themedFocusOutlines = theme.canvas.palette.common.focusOutline;
+    return {
+      '&:focus:not([disabled])': {
+        borderColor: themedFocusOutlines,
+        boxShadow: `inset 0 0 0 1px ${themedFocusOutlines}`,
+        outline: 'none',
+      },
+    };
+  },
```

Comment:
I tried to move this bit of code down below rather than having two places where theme is being called, however, it wasn't adding the focus state to the select, any idea why?

Reply:
Hmm, my guess would be the focused button styles weren't being applied somewhere. Should be working now though with the recent changes.